src/components/OSCALSystemImplementationInventoryItems.js

```diff
@@ -0,0 +1,173 @@
+import React from "react";
+import { styled } from "@mui/material/styles";
+import Typography from "@mui/material/Typography";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell, { tableCellClasses } from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+
+const SmallTableCell = styled(TableCell)`
+  text-align: right;
+  padding: 0.75em 0.75em;
+`;
+
+const OSCALSystemImplementationTableTitle = styled(Typography)`
+  flex: 1 1 100%;
+  padding-top: 1em;
+  padding-bottom: 1em;
+`;
+
+const StyledHeaderTableCell = styled(TableCell)(({ theme }) => ({
+  [`&.${tableCellClasses.head}`]: {
+    backgroundColor: theme.palette.primary.main,
+    color: theme.palette.common.white,
+  },
+  textAlign: "left",
+  minWidth: "10em",
+}));
+
+const StyledTableRow = styled(TableRow)(({ theme }) => ({
+  // Use hover color for even numbered rows
+  "&:nth-of-type(even)": {
+    backgroundColor: theme.palette.action.hover,
+  },
+}));
+
+const OSCALPropertiesSubDataHeader = styled(TableCell)`
+  text-transform: capitalize;
+  white-space: nowrap;
+  padding: 0.75em 0.75em;
+`;
```

Comment:
It seems like some of these are pretty consistent across the newly added files; can we extract these to another `OSCALSystemImplementationTableStyles.js` file or something?